.travis.yml

```diff
@@ -42,4 +42,7 @@ matrix:
     - env: JYTHON=true; YTDL_TEST_SET=download
 before_install:
   - if [ "$JYTHON" == "true" ]; then ./devscripts/install_jython.sh; export PATH="$HOME/jython/bin:$PATH"; fi
-script: ./devscripts/run_tests.sh
+script:
+  - pip install "flake8; python_version >= '2.7' and python_version != '3.0' and python_version != '3.1' and python_version != '3.2' and  python_version != '3.3'"
+  - if [ -x "$(command -v flake8)" ]; then flake8 . ;fi
```

Comment:
This must be executed one single time in a separate job. Running this in each job is just useless.

Reply:
fixed